web/src/components/Footer.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Box,
+  Flex,
+  IconButton,
+  Text,
+  useColorMode,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import { BsLinkedin } from "react-icons/bs";
+import { VscGithub, VscGithubInverted, VscTwitter } from "react-icons/vsc";
```

Comment:
what do you think of using only `chackra/icons` and remove `react-icons` since it seems they are not used anywhere else

Reply:
I am using chakra icons this is being passed as prop to chakra Icon component as chakra do not have this icons